python/src/zapata/rest/__init__.py

```diff
@@ -91,14 +91,21 @@ def dispatch(self, method, topic, envelope, context):
 
         endpoint, variables = self.path_discover(self.web_endpoint, topic, self.data_layer_endpoint)
 
+        params = envelope.get('params')
```

Comment:
params = zpt.merge(envelope.get('params'), self.variable_resolver(self.data_layer_params, variables)

Reply:
The bridge doesn't accept merge an empty object with a dictionary, so I needed to test before to have sure that there is an object before to merge.

Reply:
Bad design, sorry. The merge function MUST accept nil values on either one of the parameters.
Already corrected, will be include in the next release